releases/release-1.31/release-notes/maps/pr-123638-map.yaml

```diff
@@ -1,4 +1,5 @@
 pr: 123638
 releasenote:
-  text: Graduated MatchLabelKeys/MismatchLabelKeys feature in PodAffinity/PodAntiAffinity to Beta.
+  text: Graduated MatchLabelKeys/MismatchLabelKeys feature in PodAffinity/PodAntiAffinity
```

Comment:
```suggestion
  text: Graduated MatchLabelKeys/MismatchLabelKeys feature in PodAffinity/PodAntiAffinity.
```

Reply:
Sorry this is wrong, no need to update here.

Reply:
ok, thanks